File: os2skoledata-azure-ad-sync/src/main/java/dk/digitalidentity/config/modules/UsernameSettings.java
package dk.digitalidentity.config.modules;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class UsernameSettings {
	private UsernameStandard usernameStandard;
	private String usernamePrefix;
	private int randomStandardLetterCount = 4;
	private int randomStandardNumberCount = 4;
}

File: os2skoledata-ad-sync/Services/ActiveDirectory/SettingsModel/UsernameSettings.cs
namespace os2skoledata_ad_sync.Services.ActiveDirectory
{
    public class UsernameSettings
    {
        public UsernameStandardType UsernameStandard { get; set; }
        public string UsernamePrefix { get; set; }
        public int RandomStandardLetterCount { get; set; } = 4;
        public int RandomStandardNumberCount { get; set; } = 4;
    }
}